tests/unit/test_repair.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import sys
+
+import pytest
+from auditwheel.repair import copylib
+from auditwheel.tmpdirs import InTemporaryDirectory
+from auditwheel.lddtree import lddtree
+
+@pytest.fixture
+def systemlib():
+    for libdata in lddtree(sys.executable)["libs"].values():
+        # return one of the arbitrary libraries that python is linked
+        # against, like libc.so.6. any one will do.         
+        return libdata["realpath"]
```

Comment:
You can omit the for loop:
```suggestion
    # return one of the arbitrary libraries that python is linked
    # against, like libc.so.6. any one will do.         
    lib, *_ = lddtree(sys.executable)["libs"].values()
    return lib["realpath"]
```